lib/client/api.go

```diff
@@ -681,74 +696,95 @@ func (tc *TeleportClient) runShell(nodeClient *NodeClient, sessionID session.ID,
 	if tc.Stderr == nil {
 		tc.Stderr = os.Stderr
 	}
+	attachedTerm := (stdin == os.Stdin && term.IsTerminal(0))
+
+	winSize := &term.Winsize{Width: 80, Height: 25}
```

Comment:
mmm. no. I created `defaults` package to keep _configuration defaults_. It's not a place to dump every constant from every part of Teleport. Moving them to local constants also won't make code more readable, because it clearly says "this is width, this is height".